docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -71,7 +71,7 @@ to generate the `extraData` RLP string to include in the genesis file.
     besu rlp encode --from=toEncode.json
     ```    
 
-Where the `toEncode.json` file contains a list of the initial validators in ascending order. 
+Where the `toEncode.json` file contains a list of the initial validators in ascending order. Use the Besu subcommand [`public-key export-address`](../../../Reference/CLI/CLI-Subcommands.md#export-address) to write the address of a validator and copy it to the `toEncode.json` file. For example:
```

Comment:
address of a validator -> validator address 

Suggested edit to reduce wordiness

Reply:
Updated.